README.rst

```diff
@@ -276,6 +276,31 @@ defined:
 The information about the worker_id in a test is stored in the ``TestReport`` as
 well, under the ``worker_id`` attribute.
 
+
+Uniquely identifying the current test run
+-----------------------------------------
+
+*New in version 1.32.*
+
+If you need to globally distinguish one test run from others in your
+workers, you can use the ``session_uuid`` fixture:
+
+.. code-block:: python
+
+    @pytest.fixture(scope="session", autouse=True)

```

Comment:
to me thos examples looks awfully constructed, is there any more natural example?
the given example is something i would only ever implement at pytest_sessionstart hook time on the coordinating process

Reply:
Indeed, I actually use it for process synchronization. I'll update with a more real-world example.

Reply:
I added a less contrived example that's actually close to a real-world use case that I have.

Reply:
Lovely, so much more relatable